decidim-core/app/types/decidim/core/has_locale_input_sort.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    module HasLocaleInputSort
```

Comment:
Do you think is a better name?
```suggestion
    module HasLocalizedInputSort
```